### DISCUSSION TITLE
How to handle when a text box can't be found by hint mode?

### DISCUSSION THREAD

**Discussion by YaleLowe** [tridactyl/tridactyl#5093] [Q&A] [answered]
I've noticed that ChatGPT has a new text box and hint mode isn't able to highlight it anymore (using `f`). However, when we use `;;`, it's possible to get into the text box if the correct hinted element is selected. But then there are so many hinted elements surrounding the text box that it isn't immediately obvious which one will get into the text box.

Just curious if I'm missing some functionality that I should be using to better handle this sort of thing? I'm currently assuming that using `;;` is the suggested way to do it.

**Comment by bovine3dom** [comment, marked as answer]
Hi, I'd recommend writing a hint mode that exactly matches the text box as described here https://tridactyl.xyz/newsletters/tips-and-tricks/1-hint-css-selectors/

Then I'd bind that to `gi` (assuming that `gi` doesn't work out of the box, which is worth checking) using `:bindurl [URL] gi hint ...`

**Reply by YaleLowe** [reply]
Thank you very much! `gi` does indeed work out of the box. I knew there was something I missed in the documentation as I had forgotton about jumping to an input.
